.circleci/config.yml

```diff
@@ -95,12 +96,24 @@ commands:
                           export CYPRESS_AWS_SECRET_ACCESS_KEY=$(aws configure get aws_secret_access_key --profile default) ;
                           export CYPRESS_AWS_ACCESS_KEY_ID=$(aws configure get aws_access_key_id --profile default) ;
                           export CYPRESS_AWS_SESSION_TOKEN=$(aws configure get aws_session_token --profile default) ;
-                          echo "Running against $CYPRESS_ENVIRONMENT environment" ;
-                          npm install ;
-                          if [ <<parameters.trigger-context>> == "external" ] && [ <<parameters.stage>> == "production" ]
-                          then ./node_modules/.bin/cypress run --record --key 31a18a85-d0e5-4c16-a1d4-5f0eab6a7c58 -e TAGS='@Production and not @GoogleLighthouse and not @Accessibility and not @ignore and not @device' --parallel --group 'all tests'
-                          else ./node_modules/.bin/cypress run --record --key 31a18a85-d0e5-4c16-a1d4-5f0eab6a7c58 -e TAGS='not @GoogleLighthouse and not @Accessibility and not @ignore and not @device' --parallel --group 'all tests'
+                          
+                          export CYPRESS_TAGS='not @GoogleLighthouse and not @Accessibility and not @ignore' ;
+                          export PIPELINE_FILTER="$(echo << pipeline.parameters.upstream_pipeline_name >> | sed 's,https://github.com/LBHackney-IT/mtfh-frontend-,,g')"
+                          [ <<parameters.trigger-context>> == "external" ] && [ <<parameters.stage>> == "production" ] && export CYPRESS_TAGS="$CYPRESS_TAGS and @Production"
+
+                          if [ -n "$PIPELINE_FILTER" ]; then
+                            [[ $PIPELINE_FILTER == "common" ]] && export PIPELINE_FILTER="SmokeTest"
```

Comment:
I'd say `root` and `auth` MFEs are also something you wouldn't want to test in isolation.